### PR TITLE
for size under 127

### PR DESCRIPTION
**Pull request by dtynn**

### CODE REVIEW

File: padreader.go
Comment (schomatis) [inline padreader.go:17]: Extra points if you change this to just `Len64()`.

File: padreader.go
Comment (schomatis) [inline padreader.go:13]: An `UnpaddedPieceSize` smaller than 127 (actually not a multiple of) is invalid, so maybe we should test that and reject this case, but I might be missing context on the rationale of this PR.

**Review by whyrusleeping — APPROVED** [review]
I think this is correct, i'll let @magik6k click the merge button though

File: padreader.go
Comment (dtynn) [reply padreader.go:13]: correct, we should either reject them (UnpaddedPieceSizes under 127) all or let them have same behavior.

for now, `PaddedSize(32) != PaddedSize(126)`, that's what this pr is for

Comment (dtynn) [reply padreader.go:13]: besides, this function is mainly used when we have to find a suitable UnpaddedPieceSize for a piece of data of any size, so i think given 127 is fine here

Comment (schomatis) [reply padreader.go:13]: Makes sense, thanks for the clarification @dtynn!

**Review by schomatis — APPROVED** [review]

**Review by magik6k — APPROVED** [review]

**Pull request merged**
